latexrun

```diff
@@ -836,15 +836,15 @@ class LaTeX(Task):
         pages of output.
         """
         jobname = outname = None
-        for m in re.finditer(r'^Transcript written on (.*)\.log\.$', stdout,
+        for m in re.finditer(r'^Transcript written on (")?(.*)\.log(")?\.$', stdout,
```

Comment:
Since you don't need the capture group, use "? instead of (")?.  Then you don't have to change the group numbers below.